docs/jobs_usage.rst

```diff
@@ -1,23 +1,172 @@
+                resources=Resources(memory_mb=100, cpu=0.5),
+                command="sleep 30",
+            )
+        )
+
+
+The example above starts a job using ``ubuntu:latest`` public image, ``100 MB`` of
+memory, ``0.5`` CPU and executes ``sleep 30`` command inside started container.
+
+
+.. note::
+
+   After return from the call a new job is *scheduled* for execution but usually it's
+   status is *pending*.  The Neuromation Platform takes time for preparing resources for
+   a job, pulling image from registry etc.  Startup time can vary from seconds for *hot
+   start* to minutes for *cold* one.
+
+
+Check Job Status
+================
+
+After spawning a job we have :class:`JobDescription` instance that describes job status
+(and other things like executed command line).
+
+A job takes time for deployment, it can be terminated by different reasons,
+e.g. requested image name doesn't exist.
+
+The following snippet waits for job's *starting execution* or *failure*::
+
+   # job is a JobDescription given by former client.job.run() call
+
+   while True:
+       job = await client.jobs.status(job.id)
+       if job.status in (JobStatus.RUNNING, JobStatus.SUCCEEDED):
+           break
+       elif job.status == JobStatus.FAILED:
+           raise RuntimeError(f"Job {job.id} failed with {job.reason}:"
+                              f"{job.history.description}")
+       else:
+           await asyncio.sleep(1)
+
+
+Mount Neuromation Storage folders
+=================================
+
+The Neuromation Platform provides access to Neuromation storage (``storage://``) by
+*mounted folders* inside a container (*volumes* in `Docker <https://www.docker.com>`_
+glossary).
+
+
+If you have a directory ``storage:folder`` and want to mount it inside a container under
+``/var/data`` path please create a :class:`Volume` and use it in :class:`Container`
+definition::
+
+   from yarl import URL
+
+   volume = Volume(
+       storage_uri=URL("storage:folder"),
+       container_path="/var/data",
+   )
+
+   job = await client.jobs.run(
+       Container(
+           image=client.parse.remote_image("ubuntu:latest"),
+           resources=Resources(memory_mb=100, cpu=0.5),
+           command="sleep 30",
+           volumes=[volume],
+       )
+   )
+
+
+There is a parsing helper that can construct a :class:`Volume` instance from a string in
+format that is used in :term:`CLI`::
+
+   volume = client.parse.volume("storage:folder:/var/data")
+
+.. seealso:: :ref:`storage-usage` for the storage manipulation API.
+
+
+Kill a Job
+==========
+
+Use :meth:`Jobs.kill` for enforcing job to stop::
+
+   await client.jobs.kill(job.id)
 
 
+Expose job's TCP ports locally
+==============================
+
+Sometimes there is a need to access TCP endpoints exposed by a job executed on the
+Neuromation Platform from local workstation.
+
+For example, you've started a gRPC server inside a container on TCP port ``12345`` and
```

Comment:
I'd like to provide a real (or real-like) example here for binary TCP protocol (HTTP port can be exposed by `container.http.port` property without port forwarding.
Ideas?